mcp-servers/slack/README.md

```diff
@@ -0,0 +1,121 @@
+
+2. `slack_post_message`
+   - Post a new message to a Slack channel
+   - Required inputs:
+     - `channel_id` (string): The ID of the channel to post to
+     - `text` (string): The message text to post
+   - Returns: Message posting confirmation and timestamp
+
+3. `slack_reply_to_thread`
+   - Reply to a specific message thread
+   - Required inputs:
+     - `channel_id` (string): The channel containing the thread
+     - `thread_ts` (string): Timestamp of the parent message
+     - `text` (string): The reply text
+   - Returns: Reply confirmation and timestamp
+
+4. `slack_add_reaction`
+   - Add an emoji reaction to a message
+   - Required inputs:
+     - `channel_id` (string): The channel containing the message
+     - `timestamp` (string): Message timestamp to react to
+     - `reaction` (string): Emoji name without colons
+   - Returns: Reaction confirmation
+
+5. `slack_get_channel_history`
+   - Get recent messages from a channel
+   - Required inputs:
+     - `channel_id` (string): The channel ID
+   - Optional inputs:
+     - `limit` (number, default: 10): Number of messages to retrieve
+   - Returns: List of messages with their content and metadata
+
+6. `slack_get_thread_replies`
+   - Get all replies in a message thread
+   - Required inputs:
+     - `channel_id` (string): The channel containing the thread
+     - `thread_ts` (string): Timestamp of the parent message
+   - Returns: List of replies with their content and metadata
+
+
+7. `slack_get_users`
+   - Get list of workspace users with basic profile information
+   - Optional inputs:
+     - `cursor` (string): Pagination cursor for next page
+     - `limit` (number, default: 100, max: 200): Maximum users to return
+   - Returns: List of users with their basic profiles
+
+8. `slack_get_user_profile`
+   - Get detailed profile information for a specific user
+   - Required inputs:
+     - `user_id` (string): The user's ID
+   - Returns: Detailed user profile information
+
+## Setup
+
+### Setting up Slack OAuth
+
+1. Create a Slack App:
+   - Visit the [Slack Apps page](https://api.slack.com/apps)
+   - Click "Create New App"
+   - Choose "From scratch"
+   - Name your app and select your workspace
+
+2. Configure Bot Token Scopes:
+   Navigate to "OAuth & Permissions" and add these scopes:
+   - `channels:history` - View messages and other content in public channels
+   - `channels:read` - View basic channel information
+   - `chat:write` - Send messages as the app
+   - `reactions:write` - Add emoji reactions to messages
+   - `users:read` - View users and their basic information
+
+4. Install App to Workspace:
+   - Click "Install to Workspace" and authorize the app
+   - Save the "Bot User OAuth Token" that starts with `xoxb-`
+
+5. Get your Team ID (starts with a `T`) by following [this guidance](https://slack.com/help/articles/221769328-Locate-your-Slack-URL-or-ID#find-your-workspace-or-org-id)
+
+### Setting up on OCP
+#### Usage with Quay and Podman
+
```

Comment:
Add a line here to say something like "We will first need to build an image for the Slack MCP server from the `Containerfile`"